samples/csharp/end-to-end-apps/MulticlassClassification-GitHubLabeler/GitHubLabeler/GitHubLabelerConsoleApp/DataProcessor.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.ML;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Transforms.Categorical;
+using Microsoft.ML.Transforms.Text;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GitHubLabeler
+{
+    public class DataProcessor
+    {
+        public IEstimator<ITransformer> DataProcessPipeline { get; private set; }
+
+        public DataProcessor(MLContext mlContext)
+        {
+            // Configure data transformations in the Process pipeline
+
+            DataProcessPipeline = new ValueToKeyMappingEstimator(mlContext, "Area", "Label")
+                                  .Append(new TextFeaturizingEstimator(mlContext, "Title", "TitleFeaturized"))
+                                  .Append(new TextFeaturizingEstimator(mlContext, "Description", "DescriptionFeaturized"))
```

Comment:
Recommend explicitly setting the ngram lengths to bigram+trichargrams:
https://github.com/dotnet/machinelearning/blob/7e5ee22c30d1184afa93a77a34de1aa7bb1e909b/test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/SentimentPredictionTests.cs#L51-L52

This both lets users know the parameters exist, and encourages users to modify them.

Reply:
For now, I think we prefer to keep the code simpler in most of the samples. But feel free to submit a PR about this point. We could add it as another possible more advanced and accurate configuration. 👍

Reply:
The added Bigram+Trichar isn't to have a better accuracy of the sample's dataset, simply the pedagogical route to have users make better models for their dataset. Users moving the n-gram lengths is often a quick route to gains, and the further hope of get users thinking about feature engineering (and not just learners + hyperparameters). 

You're right, it's a trade-off.

Reply:
Feel free to add a small PR with this alternate code including n-gram lengths. At least, we could show it commented as another alternative for the dev/user to explore.